pandas/tests/series/test_analytics.py

```diff
@@ -1330,6 +1330,12 @@ def test_duplicate_keep_all_ties(self):
         expected = Series([6, 7, 7, 7, 7], index=[7, 3, 4, 5, 6])
         assert_series_equal(result, expected)
 
+    def test_boolean(self):
+        s = Series([True, False])
```

Comment:
Reference issue as a comment beneath the `def test_boolean...` line.

Reply:
ok. will do.